app/models/bot/alegre.rb

```diff
@@ -115,63 +163,89 @@ def self.request_api(method, path, params = {})
     http.use_ssl = uri.scheme == 'https'
     begin
       response = http.request(request)
-      JSON.parse(response.body)
+      puts response.body
```

Comment:
Please remove this line :D

Reply:
Or replace by `Rails.logger.info`, with some prefix if it's import to log this output